Resources/Textures/Objects/Materials/materials.rsi/meta.json

```diff
@@ -58,6 +58,9 @@
         {
             "name": "tinbar_single"
         },
+        {
+            "name": "plasma_ore"
+        },
         {
             "name": "phoron_ore"
         },
```

Comment:
smh my head

Reply:
the old texture is still there ye, should I remove it?